public/wp-content/themes/ColoredCow/src/scss/style.scss

```diff
@@ -11,27 +11,56 @@ Text Domain: coloredcow
 @import 'variables';
 @import 'mixin';
 @import 'fonts';
-
 // sass code for image alignment with content
-.alignright{
-	float: right;
-	&img{
-		margin: 0 0 1em 1em;
-	}
+.alignright {
+    float: right;
+    &img {
+        margin: 0 0 1em 1em;
+    }
+}
+
+.alignleft {
+    float: left;
+    &img {
+        margin: 0 1em 1em 0;
+    }
 }
-.alignleft{
-	float: left;
-	&img{
-		margin: 0 1em 1em 0;
-	}
+
+.aligncenter {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    &img {
+        display: block;
+    }
+}
+
+.page-title {
+    font-family: 'Gloria Hallelujah', cursive;
```

Comment:
can we create these families in fonts.scss and use them here with @extend property?
Create a class for font family and extend it here.